code/modules/power/supermatter/supermatter.dm

```diff
@@ -476,6 +476,8 @@
 /obj/machinery/power/supermatter_shard/attack_paw(mob/user)
 	return attack_hand(user)
 
+/obj/machinery/power/supermatter_shard/attack_animal(mob/living/simple_animal/user)
+	return attack_hand(user)
```

Comment:
Don't treat a built in event (attack_hand) like a regular proc, refactor the dusting code into a seperate from and call that from attack_hand/paw/animal